src/components/MapImageControl.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="map-image-control">
+    <button
+      title="Export as image"
+      class="map-image-control__button leaflet-bar"
+      :disabled="exporting"
+      @click.prevent.stop="showPopUp = true"
+    >
+      <img :src="saveIcon" alt="" />
+    </button>
+    <!-- the modal needs to be rendered outside of the map, the portal takes care of that -->
+    <portal to="portal-target">
+      <pop-up v-if="showPopUp" @close="showPopUp = false">
+        <form action="" submit.prevent class="map-image-control__form">
+          <div class="control-group">
+            <h3 class="control-label">Interface</h3>
+
+            <label>
+              <input type="checkbox" v-model="showControls" />
+              Toon controls
+            </label>
+          </div>
+
+          <div class="control-group">
+            <h3 class="control-label">Formaat</h3>
+
+            <div v-for="size in sizes" :key="size.name">
+              <label>
+                <input type="radio" :value="size.className" v-model="exportSize">
+                {{ size.name }}
+              </label>
+            </div>
+          </div>
+
+          <div class="control-group">
+            <h3 class="control-label">Locatie</h3>
+
+            <p class="map-image-control__location">Coördinaten linksboven: {{ origin.join(', ') }}</p>
+            <p>Zoomniveau: {{ zoomLevel }}</p>
+          </div>
+
+          <button class="btn primary" @click.prevent.stop="exportAsImage">Exporteren als afbeelding</button>
+        </form>
+
+        <!-- TODO: add coordinates of top left corner -->
```

Comment:
Did you plan to add this?

Reply:
Forgot to remove the comment, the functionality is in there